### PR TITLE
Quadratic joystick

### PR DESCRIPTION
**Pull request by 3LucasZ**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java [hunk 1]
```diff
@@ -372,6 +372,14 @@ public void periodic() {
     }
   }
 
+  public static double joystickToSpeed(double joystickValue, double maxValue) {
+    if (FeatureFlags.kQuadraticDrive) {
+      return Math.signum(joystickValue) * joystickValue * joystickValue * maxValue;
```
Comment (realjoshuau) [inline src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java:377]: explain the math here

**Review by realjoshuau — APPROVED** [review]
lgtm, explain the math here and merge

**Review by realjoshuau — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java (hunk 1 above)
Comment (3LucasZ) [reply src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java:377]: Math behind quadratic joystick 
Screenshot 2024-02-04 at 6 00 14 PM

Comment (realjoshuau) [reply src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java:377]: ok fire

**Pull request merged**